includes/Renderer/Stories/Carousel_Renderer.php

```diff
@@ -121,10 +121,13 @@ public function render( array $args = [] ): string {
 					?>
 					<div class="web-stories-list__carousel <?php echo esc_attr( $this->get_view_type() ); ?>" data-id="<?php echo esc_attr( 'carousel-' . $this->instance_id ); ?>">
 						<?php
-						foreach ( $this->stories as $story ) {
-							$this->render_single_story_content();
-							$this->next();
-						}
+						array_map(
```

Comment:
I thought you were not a fan of `array_map` and preferred `foreach` instead?

What's the context for this change?

Reply:
https://github.com/slevomat/coding-standard/blob/master/doc/variables.md#slevomatcodingstandardvariablesunusedvariable

Reply:
No issue, just a question.

Reply:
Mark as resolved.